geopandas/tools/sjoin.py

```diff
@@ -28,7 +28,20 @@ def sjoin(
         Suffix to apply to overlapping column names (left GeoDataFrame).
     rsuffix : string, default 'right'
         Suffix to apply to overlapping column names (right GeoDataFrame).
+    """
+    _basic_checks(left_df, right_df, how, op, lsuffix, rsuffix)
+
+    result = _geom_predicate_query(left_df, right_df, op)  # consume op param
```

Comment:
Can we not call this intermediate df `results`? It can become confusing why there are other steps after results.

Reply:
I renamed to `indices`